### ISSUE TITLE
[Warning] S.M.A.R.T. output reading error: exit status 4

### ISSUE THREAD

**Issue by marcust** [Sheridan/smartctl_exporter#8]
I am running the exporter on a Raspberry PI 4 that is my fileserver, it has 3 external HDs connected to it. 

For two of them I get
` [Warning] S.M.A.R.T. output reading error: exit status 4` 
and for one I get 
`[Warning] S.M.A.R.T. output reading error: exit status 2` (this is actually a WD Elements, so a very basic disk that does not have a lot of features the big external drives have)

As per #2 additional information:
```
smartctl --version
smartctl 7.1 2019-12-30 r5022 [armv7l-linux-5.4.42-v7l+] (local build)
Copyright (C) 2002-19, Bruce Allen, Christian Franke, www.smartmontools.org
```
(Installed from the Debian Buster Backports)

Output of `smartctl --json --xall /dev/sda > /tmp/sda.json`
[sda_json.txt](https://github.com/Sheridan/smartctl_exporter/files/4675541/sda_json.txt)
[sdb_json.txt](https://github.com/Sheridan/smartctl_exporter/files/4675542/sdb_json.txt)
[sdc_json.txt](https://github.com/Sheridan/smartctl_exporter/files/4675543/sdc_json.txt)

`sda` is the WD Elements, I guess it just does not support any SMART output? 

The other two though should not lead to an error I guess?

**Comment by Haxprox** [comment]
Behaves similarly on the same Debian version:

```
/usr/bin/smartctl_exporter --verbose --debug
[Verbose] Read options from /etc/smartctl_exporter.yaml

[Debug] Parsed options: {{[::1]:9633 /metrics %!s(bool=false) /usr/sbin/smartctl 120s 2m0s [/dev/sda /dev/sdb /dev/sdc]}}
[Debug] Collecting S.M.A.R.T. counters, device: /dev/sda
[Warning] S.M.A.R.T. output reading error: exit status 100
[Verbose] Collecting metrics from /dev/sda: Seagate Surveillance, ST2000VX002-1AH166
[Debug] Collecting S.M.A.R.T. counters, device: /dev/sdb
[Warning] S.M.A.R.T. output reading error: exit status 4
[Verbose] Collecting metrics from /dev/sdb: Seagate Surveillance, ST2000VX002-1AH166
[Debug] Collecting S.M.A.R.T. counters, device: /dev/sdc
[Warning] S.M.A.R.T. output reading error: exit status 2
[Verbose] Collecting metrics from : , 
[Info] Starting on [::1]:9633/metrics
^C

```

**Comment by mr-yaky** [comment]
I get the same exit status error:
```
Aug 26 17:47:51 host systemd[1]: Started Prometheus S.M.A.R.T Exporter.
Aug 26 17:47:51 host smartctl-exporter[19855]: [Warning] S.M.A.R.T. output reading error: exit status 4
Aug 26 17:47:51 host smartctl-exporter[19855]: [Warning] S.M.A.R.T. output reading error: exit status 4
Aug 26 17:47:51 host smartctl-exporter[19855]: [Info] Starting on 0.0.0.0:9633/metrics
```
Also I used that `smartctl_device_smartctl_exit_status` metric for  a Prometheus alert and now always get the alerts for some specific SSD model name: Micron_1100_MTFDDAK512TBN 

Did anyone solve that problem ? Any ideas ?

**Comment by Sheridan** [comment]
Possibly fixed in version 0.6
